.github/workflows/single-host-tests.yaml

```diff
@@ -0,0 +1,34 @@
+#
+#  Copyright (c) 2012-2020 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+#
+
+name: Testing single host nightly deployment
```

Comment:
Let's name it:
`Testing latest changes [single-host/gateway]` and `Testing latest changes [single-host/native]`
Maybe somebody have better ideas.